### DISCUSSION TITLE
How to get a recipient if to_recipients turns out to be None

### DISCUSSION THREAD

**Discussion by Bart-Jan** [ecederstrand/exchangelib#1162] [Q&A]
Hi,

using exchangelib 4.90, with Python 3.10, I am accessing an Exchange Server with EWS-Protocoll 2016.
For some emails in the account I am supposed to work with `email.to_recipients` get set to `None`.
The part of the code I am using 
```
         for m in a_subfolder.all().only("to_recipients", "subject"):
            if isinstance(m, Message):
               print('         Subject is . ', m.subject, ' recipients are : ', m.to_recipients)
```

However if I use the OutLook webclient for the same account recipient shows as valid email address for those emails.
Anybody have a tip on how I can get to the same recipient the webclient shows using Python/exchangelib? I am supposed to take action depending on (a part of) the recipient.

TIA,

Bart-Jan.

**Comment by ecederstrand** [comment]
Was this email sent from the Exchange account, or received by the Exchange account. If it's the latter, then you may need to dive into the MIME data to find out who the email was sent to.
